operator-pipeline-images/operatorcert/entrypoints/fbc_onboarding.py

```diff
@@ -311,11 +234,12 @@ def render_fbc_from_template(operator: Operator) -> None:
             "opm",
             "alpha",
             "render-template",
-            "composite",
-            "-f",
-            COMPOSITE_TEMPLATE_CATALOGS,
-            "-c",
-            COMPOSITE_TEMPLATE_CONTRIBUTIONS,
+            "basic",
+            "-o",
+            "yaml",
+            os.path.join(operator.root, CATALOG_TEMPLATES_DIR, f"v{version}.yaml"),
+            ">",
+            f"../../catalogs/v{version}/catalog.yaml"
```

Comment:
I think you're missing an operator name in the output path. Based on the design it should be `../../catalogs/v{version}/{operator-name}/catalog.yaml`